bundles/scripting/sightly/js-use-provider/pom.xml

```diff
@@ -70,6 +70,7 @@
                     <instructions>
                         <Sling-Bundle-Resources>/libs/sling/sightly/js;path:=/SLING-INF/libs/sling/sightly/js</Sling-Bundle-Resources>
                         <Require-Capability>io.sightly; filter:="(&amp;(version&gt;=1.0)(!(version&gt;=2.0)))"</Require-Capability>
+                        <Import-Package>org.mozilla.javascript;version="[1.1,2)",*</Import-Package>
```

Comment:
@kwin, I'll add the comment in the `pom.xml` file.

The thing is that the JS Use Provider was using the same Rhino version as the `org.apache.sling.scripting.javascript` bundle was embedding until recently. However, the provider was not depending on the JavaScript bundle, which meant that the `org.mozilla.javascript` package was imported without an import range.

Since SLING-6668, the `org.mozilla.javascript` package is exported by the `servicemix` bundle that @oliverlietz brought into the Sling launchpad. However, that exports version 1.7, while the JS Provider works ok with anything in the [1.1, 2) range, hence the manually defined import range.